.storybook/seed-fake.js

```diff
@@ -104,6 +104,25 @@ export const seedFake = (db) => {
     is_managed: true,
   })
 
+  db.addAccessCode({
+    device_id: device1.device_id,
+    workspace_id: ws2.workspace_id,
+    created_at: '2023-05-19T03:11:10.000',
+    name: "Bob's Front Door Code",
+    code: '3333',
+    common_code_key: null,
+    type: 'ongoing',
+    status: 'set',
+    errors: [
+      {
+        error_code: 'failed_to_set_on_device',
+        message: 'Code not set',
```

Comment:
```suggestion
        message: 'An error occurred when we tried to set the access code on the device. We will continue to try and set the code on the device in case the error was temporary.',
```